ui/dashboards/src/components/GridLayout/GridContainer.tsx

```diff
@@ -35,6 +35,7 @@ export function GridContainer(props: GridContainerProps) {
         // (see https://github.com/react-grid-layout/react-grid-layout/issues/103)
         '& .react-grid-item.cssTransforms': { transitionProperty: isFirstRender ? 'none' : 'transform' },
       }}
+      data-testid="panel-group"
```

Comment:
A bunch of these `data-testid`s wouldn't be needed if we had better a11y handling for the panel groups, but I didn't want to fix that in what is already a pretty big testing PR, so I started with safer (from a code change perspective) test attributes for now.